javascript/builtins/WeakMap.json

```diff
@@ -402,7 +402,7 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": false
+                "version_added": "20.2.0"
```

Comment:
```suggestion
                "version_added": "20.1.0"
```

Reply:
20.1.0 is correct, but the system said that was not an allowed Node version, so I chose the first one in the "allowed" list. Please let me know what I should do instead to get 20.1.0 allowed.

Reply:
Oh yes, sorry, I should have checked! The file that controls the allowlist is https://github.com/mdn/browser-compat-data/blob/main/browsers/nodejs.json. Would you be happy to add a new entry for the 20.1.0 release there?

Reply:
I updated the file and put the version back to 20.1.0

Reply:
Excellent, thank you! 👍